jobbergate-api/jobbergate_api/apps/applications/routers.py

```diff
@@ -75,18 +75,20 @@ async def applications_create(
     "/applications/{application_id}/upload",
     status_code=status.HTTP_201_CREATED,
     description="Endpoint for uploading application files.",
-    dependencies=[Depends(guard.lockdown(Permissions.APPLICATIONS_EDIT))],
 )
 async def applications_upload(
     application_id: int = Query(..., description="id of the application for which to upload a file"),
     upload_files: List[UploadFile] = File(
         ..., media_type="text/plain", description="The application files to be uploaded"
     ),
     content_length: int = Header(...),
+    token_payload: TokenPayload = Depends(guard.lockdown(Permissions.APPLICATIONS_EDIT)),
 ):
     """
     Upload application files using an authenticated user token.
     """
+    application = await _fetch_and_verify_ownership(application_id, token_payload)
```

Comment:
looking at this line before looking at the other changes is a little bit strange. I mean, there's a call to a function whose name is `_fetch_and_verify_ownership`, so I expect the code you fetch the ownership and verify it, simple. However, there's an assignment for a variable whose name is `application`. I feel some lack of transparency on what's going on in this line.

Reply:
I guess the function could be named `_fetch_application_and_verify_ownership()`. A little verbose but not a problem.

Reply:
Addressed in commit d6e1add